.github/workflows/release.yaml

```diff
@@ -0,0 +1,51 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      # See https://github.com/helm/chart-releaser-action/issues/6
+      - name: Install Helm
+        run: |
+          curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get
+          chmod 700 get_helm.sh
+          ./get_helm.sh
+          helm init --client-only
+
+      - name: Add dependency chart repos
+        run: |
+          helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+          helm repo add elastic https://helm.elastic.co
+
+      - name: Create kind cluster
+        uses: helm/kind-action@master
+        with:
+          installLocalPathProvisioner: true
+
+      - name: Run chart-testing (lint)
+        uses: helm/chart-testing-action@master
+        with:
+          command: lint
+
+      - name: Run chart-testing (install)
+        uses: helm/chart-testing-action@master
+        with:
+          command: install
```

Comment:
I forked the branch of this PR, and made some changes to the manifests.
But, these steps output `No chart changes detected.`
(See: https://github.com/skomma/helm-charts-1/actions)

Are these steps correct?

Reply:
Confirmed not working as expected...
I'll look into this issue at the weekend.